homeassistant/components/webostv/media_player.py

```diff
@@ -394,7 +394,24 @@ async def async_play_media(
         """Play a piece of media."""
         _LOGGER.debug("Call play media type <%s>, Id <%s>", media_type, media_id)
 
-        if media_type == MediaType.CHANNEL:
+        if media_type == MediaType.VIDEO:
+            if (source_dict := self._source_list.get("Media Player")) is not None:
```

Comment:
We should probably log or raise error if it fails, try to look at other media players for how to handle missing source